src/services/copyConfig/emails/emailConfiguration.js

```diff
@@ -50,13 +49,34 @@ class EmailConfiguration {
     stringToJson(response, 'context')
     return Array.isArray(response) ? response : [response]
   }
-
+  #cleanUrl(options, response) {
+    for (let branch of options) {
+      let filter = branch.branches.map((obj) => obj.value)
+      if (!filter[0]) {
+        this.cleanLink(response, branch)
+        console.log('in', filter)
+      }
+    }
+  }
+  deleteLink(response, property, branch) {
+    delete response[branch][property]
+  }
+  cleanLink(response, branch) {
+    if (branch.id === 'passwordReset') {
+      this.deleteLink(response, 'resetURL', branch.id)
+    }
+    if (branch.id === 'preferencesCenter') {
+      this.deleteLink(response, 'redirectURL', branch.id)
+    }
+  }
   getEmail() {
     return this.#email
   }
 
   async copyEmailTemplates(destinationSite, destinationSiteConfiguration, response, options) {
     const promises = []
+    const collection = options.options.branches.filter((obj) => Array.isArray(obj.branches))
+    this.#cleanUrl(collection, response)
     for (const templateInfo of options.getOptions().branches) {
       if (templateInfo.value) {
         promises.push(this.#copyEmailTemplate(destinationSite, templateInfo.name, destinationSiteConfiguration, response))
```

Comment:
The idea of removing the link on this class is good. To simplify consider adding this code here:
if(templateInfo.name == 'passwordReset' && templateInfo.branches[0].value) {
   this.deleteLink(response, 'resetURL', branch.id)
}
it seems simpler than all the code added.